consensus/spos/debug/equivalentMessagesDebugger.go

```diff
@@ -2,67 +2,120 @@ package debug
 
 import (
 	"fmt"
+	"sync"
 
+	"github.com/multiversx/mx-chain-core-go/core/check"
+	"github.com/multiversx/mx-chain-core-go/data"
 	"github.com/multiversx/mx-chain-core-go/display"
-	"github.com/multiversx/mx-chain-go/consensus"
+	"github.com/multiversx/mx-chain-go/consensus/spos"
+	"github.com/multiversx/mx-chain-go/sharding"
 	logger "github.com/multiversx/mx-chain-logger-go"
 )
 
 var log = logger.GetOrCreate("debug/equivalentmessages")
 
+type proofsPoolHandler interface {
```

Comment:
why not use the main exported interface?

Reply:
changed to used the main one